### ISSUE TITLE
chart.yaml missing versions on bitnami

### ISSUE THREAD

**Issue by iburrows** [sentry-kubernetes/charts#847]
Hi, 
for various dependencies in the chart.yaml it seems like they are deprecated.
When I try to `helm dependency update` with v17.0.0 I receive 
```
error: can't get a valid version for repositories kafka, zookeeper, rabbitmq, postgresql. Try changing the version constraint in Chart.yaml
```
In the chart.yaml kafka is at 16.3.2 and if I try to pull the chart by selecting a version
```
helm pull artifactory-bitnami/kafka --version=
17.2.3  17.2.6  18.0.2  18.0.4  18.0.6  18.0.8  18.1.2  18.2.0  18.3.1  18.4.1  18.4.3  18.5.0  19.0.1  19.1.0  19.1.2  19.1.4  20.0.0  20.0.2  20.0.4  20.0.6  20.1.1  21.0.1  21.1.1  21.3.0  21.4.0  
17.2.5  18.0.0  18.0.3  18.0.5  18.0.7  18.1.1  18.1.3  18.3.0  18.4.0  18.4.2  18.4.4  19.0.0  19.0.2  19.1.1  19.1.3  19.1.5  20.0.1  20.0.3  20.0.5  20.1.0  21.0.0  21.1.0  21.2.0  21.3.1  21.4.1
```
Those are the only versions available.
Has anyone else had this issue? Where can I locate the versions? I am using https://charts.bitnami.com/bitnami

**Comment by Agalin** [comment]
Bitnami started removing old charts last year.

See:
https://github.com/bitnami/charts/pull/10530
https://github.com/bitnami/charts/pull/14024

So this chart must be updated.

**Comment by spectrogram** [comment]
I pushed a fix for this in 17.10.1: https://github.com/sentry-kubernetes/charts/blob/9fe1b754628e46215d74d28171221ed7bec5513d/sentry/Chart.yaml

**Comment by iburrows** [comment]
Thanks used the pre-2022 charts branch.
